DockerOrchestration/ABCIApp/Dockerfile.Dummy

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8
```

Comment:
I'm all for having them run in separate processes. 

> since we can orchestrate multiple ABCI apps in front of a single core node.

I don't think this makes sense yet. Tendermint isn't set up for it.